hnn_core/__init__.py

```diff
@@ -1,10 +1,7 @@
-from .utils import load_custom_mechanisms
-
-load_custom_mechanisms()
```

Comment:
could you try building the documentation? Does it complete without throwing any errors? Do the html files build? I am still struggling to install mpi4py so I can't check unfortunately.

Reply:
Yeah, building the documentation worked for me.